--- rungms.orig	2013-09-01 13:39:17.547999955 +0900
+++ rungms	2013-09-01 13:39:03.079999954 +0900
@@ -60,9 +60,9 @@
 #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
 #
 set TARGET=sockets
-set SCR=/scr/$USER
-set USERSCR=~$USER/scr
-set GMSPATH=/u1/mike/gamess
+set SCR=$HOME/scr
+set USERSCR=$HOME/scr
+set GMSPATH=$HOME/share/gamess
 #
 set JOB=$1      # name of the input file xxx.inp, give only the xxx part
 set VERNO=$2    # revision number of the executable created by 'lked' step
@@ -269,6 +269,17 @@
       set NNODES=1
       set HOSTLIST=(`hostname`)
    endif
+   if ($NCPUS >= 1) then
+      set NNODES=$NCPUS
+      set HOST=(`hostname`)
+      set HOSTLIST=$HOST
+      @ CPU=1
+      while ($CPU < $NNODES)
+         set HOSTLIST=($HOSTLIST $HOST)
+         @ CPU++
+      end
+      unset $CPU
+   endif
 #
 #       2. This is an example of how to run on a multi-core SMP enclosure,
 #          where all CPUs (aka COREs) are inside a -single- NODE.
@@ -308,8 +319,7 @@
             set HOSTLIST=(`hostname`:cpus=$NCPUS)
             breaksw
          default:
-            echo I do not know how to run this node in parallel.
-            exit 20
+            breaksw
       endsw
    endif
 #
